.project

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <projectDescription>
-	<name>RINGMesh</name>
+	<name>RINGMeshCartesian</name>
```

Comment:
No, this is still RINGMesh project! ;)

Reply:
Yes, that is due to an eclipse rename, i'll change it right away ...